content/blog/grants-and-language-update/contents.lr

```diff
@@ -0,0 +1,52 @@
+title: Extended grant support, ORCID and language field.
+---
+author: knowak
+---
+body:
+Today, we are introducing three additions to Zenodo:
+
+1. Extended grant support
+2. Language field
+3. ORCID for authors
+
+## Extended grant support (powered by OpenAIRE)
+
+We are expanding our grants database with over 620,000 grants from 8 new funders such as National Science Foundation (US) and Wellcome Trust (UK) - all thanks to [OpenAIRE](https://openaire.eu/)'s ever growing grants database.
+
+![Zenodo OpenAIRE grants database](grants.png)
+
+So far we have been supporting only grants from the European Commission (FP7 and Horizon 2020). Today our dataset contains grants from the following funders:
+
+* National Science Foundation (USA) - 497646 grants
+* European Commission (EU) - 39409 grants
+* Foundation for Science and Technology (Portugal) - 37277 grants
+* National Health and Medical Research Council (Australia) - 24354 grants
+* Netherlands Organisation for Scientific Research (The Netherlands) - 24180 grants
+* Australian Research Council (Australia) - 23011 grants
+* Wellcome Trust (United Kingdom) - 12196 grants
+* Ministry of Science and Education (Croatia) - 2120 grants
+* Ministry of Education, Science and Technological Development (Serbia) - 777 grants
+
+which means that our grants database grew from close **40,000** to over **660,000** grants! We wouldn't be able to do that if it wasn't for the OpenAIRE team and their hard work in collecting, maintaining and distributing the grants database for the benefit of Open Science!
```

Comment:
our grants database grew from ~~close~~ **nearly**
We wouldn't **have been** able to do